packages/icon-library/src/assets/table/sort-ascending.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 9 14">
```

Comment:
Could we not use these ones already in the library?

https://material.io/resources/icons/?search=down&icon=arrow_drop_down&style=round
https://material.io/resources/icons/?search=up&icon=arrow_drop_up&style=round

Maybe they aren't fit for purpose?